emcc.py

```diff
@@ -3629,7 +3629,7 @@ def consume_arg_file():
     elif check_flag('--jcache'):
       logger.error('jcache is no longer supported')
     elif check_arg('--cache'):
-      config.CACHE = os.path.normpath(consume_arg())
```

Comment:
Do you use this flag?

Reply:
No, but couple of the tests do, so invoking subprocess with a different working directory broke them.

Reply:
Although I now wonder if environment variable `EM_CACHE` (and maybe some others like `EM_CONFIG`, `EMCC_TEMP_DIR` etc?) might be subject to the same issue, just untested? Do people set those variables to relative paths / is it officially supported?

Reply:
Thoughts on this one - should I worry about other relative paths?

Reply:
Its hard to imagine folks using relative paths in EM_CACHE and EM_TEMP_DIR environment variables.   

I wouldn't worry about that case.

Reply:
Okay, fingers crossed.